membership-common/src/main/scala/com/gu/memsub/subsv2/Plan.scala

```diff
@@ -217,10 +216,9 @@ object FrontendId {
   case object Quarterly extends FrontendId { val name = "Quarterly" }
   case object Yearly extends FrontendId { val name = "Yearly" }
   case object Introductory extends FrontendId { val name = "Introductory" }
-  case object Free extends FrontendId { val name = "Free" }
```

Comment:
frontend id is a special field in the catalog so that we can distinguish between different rateplans in the same product more reliably but in a way that's agnostic between DEV/(UAT)/PROD.  In more modern codebases we just have a Map of the product rate plan IDs in the codebase.